### PR TITLE
add individual test timings

### PR DESCRIPTION
**Pull request by yaroverg**
Imported namedtuple from collections module for storing data. Added a function for printing a formatted table. Time is not stored as floats instead of integers and prints to 2 decimal places.

### CODE REVIEW

File: test_discovery.py [hunk 1]
```diff
@@ -426,6 +426,29 @@ def evaluate_entities(test_dict, resp, verbose):
     return test_single_case(test_dict, observed_entity_dict, resp, verbose)
 
 
+def print_table(timing_list, top_n=5, first_k_chars=25):
+    """ Prints a formatted table ordered by longest test timings. 
+        
+        :param timing_list: namedtuple, list of namedtuples representing testing results
+    """
```
Comment (cbrochtrup) [inline test_discovery.py]: The indentation here isn't the same as the docstrings below (`test_all`). We should probably be consistent with our formatting.

Comment (mgoldey) [reply test_discovery.py]: Agreed we should match https://www.python.org/dev/peps/pep-0257/#handling-docstring-indentation

File: test_discovery.py [hunk 2]
```diff
@@ -426,6 +426,29 @@ def evaluate_entities(test_dict, resp, verbose):
     return test_single_case(test_dict, observed_entity_dict, resp, verbose)
 
 
+def print_table(timing_list, top_n=5, first_k_chars=25):
+    """ Prints a formatted table ordered by longest test timings. 
+        
+        :param timing_list: namedtuple, list of namedtuples representing testing results
```
Comment (mgoldey) [inline test_discovery.py]:
```suggestion
    :param timing_list: namedtuple, list of namedtuples representing testing results
```

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community! We require contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) and we don't seem to have CLAs on file for these contributors to the Pull Request: (@yaroverg). In order for your PR to be reviewed and merged, please follow the directions at the link above.

Project team: please **do not merge this Pull Request** until Foundation staff have confirmed that a CLA is in place for the new contributor(s) listed above.

If you still have any questions or just don't know what to do next, please email [help@finos.org](mailto:help@finos.org). /CC @finos-admin

File: test_discovery.py [hunk 3]
```diff
@@ -426,6 +426,29 @@ def evaluate_entities(test_dict, resp, verbose):
     return test_single_case(test_dict, observed_entity_dict, resp, verbose)
 
 
+def print_table(timing_list, top_n=5, first_k_chars=25):
+    """ Prints a formatted table ordered by longest test timings.
```
Comment (yaroverg) [inline test_discovery.py]:
```suggestion
    """
    Prints a formatted table ordered by longest test timings. 
```

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community! We require contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) and we don't seem to have CLAs on file for these contributors to the Pull Request: (@yaroverg). In order for your PR to be reviewed and merged, please follow the directions at the link above.

Project team: please **do not merge this Pull Request** until Foundation staff have confirmed that a CLA is in place for the new contributor(s) listed above.

If you still have any questions or just don't know what to do next, please email [help@finos.org](mailto:help@finos.org). /CC @finos-admin

**Pull request merged**